### PR TITLE
Textile to mongo

### PR DESCRIPTION
**Pull request by migrenaa**

### CODE REVIEW

File: src/controllers/skillWallet.controller.ts [hunk 1]
```diff
@@ -333,80 +333,80 @@ export class SkillWalletController {
     }
   };
 
-  public getChat = async (req: any, res: Response) => {
-    try {
-      this.loggerService.info("params: " + JSON.stringify(req.params));
-      this.loggerService.info("req.query: " + JSON.stringify(req.query));
-      const skillWalletId = req.params.skillWalletId;
-      const recipient = req.query.recipient;
+  // public getChat = async (req: any, res: Response) => {
```
Comment (migrenaa) [inline src/controllers/skillWallet.controller.ts]: delete

File: src/controllers/skillWallet.controller.ts [hunk 2]
```diff
@@ -333,80 +333,80 @@ export class SkillWalletController {
...
-        return res
-          .status(404)
-          .send({ message: "skillWallet is a required field" });
-      if (!recipient || recipient < 0)
-        return res
-          .status(400)
-          .send({ message: "recipient is a required field" });
-      if (!text)
-        return res.status(400).send({ message: "text is a required field" });
+  // public addMessage = async (req: any, res: Response) => {
```
Comment (migrenaa) [inline src/controllers/skillWallet.controller.ts]: delete

File: src/controllers/skillWallet.controller.ts [hunk 3]
```diff
@@ -333,80 +333,80 @@ export class SkillWalletController {
...
+  // };
 
-  public getNotifications = async (req: any, res: Response) => {
-    try {
-      const skillWalletId = req.params.skillWalletId;
-      if (!skillWalletId || skillWalletId < 0)
-        return res
-          .status(404)
-          .send({ message: "skillWallet is a required field" });
+  // public getNotifications = async (req: any, res: Response) => {
```
Comment (migrenaa) [inline src/controllers/skillWallet.controller.ts]: delete

File: src/routers/skillWallet.router.ts [hunk 4]
```diff
@@ -27,16 +27,16 @@ export class SkillWalletRouter {
     this._router.get("/:skillWalletId/community/:communityAddress/membershipID", this.skillWalletController.getMembershipID);
     
     // Nonces
-    this._router.get("/:skillWalletId/nonces", this.skillWalletController.getNonceForValidation);
-    this._router.delete("/:skillWalletId/nonces", this.skillWalletController.deleteNonce);
-    this._router.post('/:skillWalletId/nonces', this.skillWalletController.generateNonce);
+    // this._router.get("/:skillWalletId/nonces", this.skillWalletController.getNonceForValidation);
```
Comment (migrenaa) [inline src/routers/skillWallet.router.ts]: we use these 3

File: src/routers/skillWallet.router.ts [hunk 5]
```diff
@@ -27,16 +27,16 @@ export class SkillWalletRouter {
...
 
     // Messages
-    this._router.get("/:skillWalletId/message", this.skillWalletController.getChat);
-    this._router.post("/:skillWalletId/message", this.skillWalletController.addMessage);
+    // this._router.get("/:skillWalletId/message", this.skillWalletController.getChat);
+    // this._router.post("/:skillWalletId/message", this.skillWalletController.addMessage);
 
     // Notifications
-    this._router.get("/:skillWalletId/notifications", this.skillWalletController.getNotifications);
+    // this._router.get("/:skillWalletId/notifications", this.skillWalletController.getNotifications);
```
Comment (migrenaa) [inline src/routers/skillWallet.router.ts]: delete

File: src/services/skillWallet.service.ts [hunk 6]
```diff
@@ -361,165 +360,165 @@ export const getCommunityDetails = async (
...
+//       .and("isValidated")
+//       .eq(false);
+//   const auths = (await threadDBClient.filter(
+//     QRCodeAuthCollection,
+//     query
+//   )) as QRCodeAuth[];
+//   return auths.map((l) => l.nonce);
+// };
+
+// export const invalidateNonce = async (
```
Comment (migrenaa) [inline src/services/skillWallet.service.ts]: we use this

File: src/services/skillWallet.service.ts [hunk 7]
```diff
@@ -361,165 +360,165 @@ export const getCommunityDetails = async (
...
+//     query
+//   )) as QRCodeAuth[];
+//   qrAuths.forEach((auth) => {
+//     auth.isValidated = true;
+//     if (tokenId) auth.tokenId = tokenId;
+//     threadDBClient.update(QRCodeAuthCollection, auth._id, auth);
+//   });
+// };
+
+// export const getChat = async (
```
Comment (migrenaa) [inline src/services/skillWallet.service.ts]: delete

File: src/services/skillWallet.service.ts [hunk 8]
```diff
@@ -361,165 +360,165 @@ export const getCommunityDetails = async (
...
+//     await addMessage(skillWalletId, recipient, undefined);
+//     const defaultMessage = (await threadDBClient.filter(
+//       ChatCollection,
+//       finalQuery
+//     )) as Chat[];
+//     return defaultMessage[0];
+//   }
+// };
+
+// export const getNotifications = async (
```
Comment (migrenaa) [inline src/services/skillWallet.service.ts]: delete

File: src/services/skillWallet.service.ts [hunk 9]
```diff
@@ -16,13 +16,11 @@ import {
...
-import {
-  ChatCollection,
-  QRCodeAuthCollection,
-  NotificationCollection,
-} from "../constants/constants";
+// import {
+//   ChatCollection,
+//   QRCodeAuthCollection,
+//   NotificationCollection,
+// } from "../constants/constants";
```
Comment (migrenaa) [inline src/services/skillWallet.service.ts]: delete

Comment (mladenig) [reply src/services/skillWallet.service.ts]: [318c7cc](https://github.com/SkillWallet/backend/pull/26/commits/318c7cc53bd84d56dbd4aa80e43fcca083fb14b1)

File: src/services/skillWallet.service.ts (hunk 8 above)
Comment (mladenig) [reply src/services/skillWallet.service.ts]: [318c7cc](https://github.com/SkillWallet/backend/pull/26/commits/318c7cc53bd84d56dbd4aa80e43fcca083fb14b1)

File: src/services/skillWallet.service.ts (hunk 7 above)
Comment (mladenig) [reply src/services/skillWallet.service.ts]: [318c7cc](https://github.com/SkillWallet/backend/pull/26/commits/318c7cc53bd84d56dbd4aa80e43fcca083fb14b1)

File: src/services/skillWallet.service.ts (hunk 6 above)
Comment (mladenig) [reply src/services/skillWallet.service.ts]: [b27fd23](https://github.com/SkillWallet/backend/pull/26/commits/b27fd237d4281800a86899ae653b949ee35e4c89)

File: src/routers/skillWallet.router.ts (hunk 5 above)
Comment (mladenig) [reply src/routers/skillWallet.router.ts]: [318c7cc](https://github.com/SkillWallet/backend/pull/26/commits/318c7cc53bd84d56dbd4aa80e43fcca083fb14b1)

File: src/routers/skillWallet.router.ts (hunk 4 above)
Comment (mladenig) [reply src/routers/skillWallet.router.ts]: [b27fd23](https://github.com/SkillWallet/backend/pull/26/commits/b27fd237d4281800a86899ae653b949ee35e4c89)

File: src/controllers/skillWallet.controller.ts (hunk 3 above)
Comment (mladenig) [reply src/controllers/skillWallet.controller.ts]: [318c7cc](https://github.com/SkillWallet/backend/pull/26/commits/318c7cc53bd84d56dbd4aa80e43fcca083fb14b1)

File: src/controllers/skillWallet.controller.ts (hunk 2 above)
Comment (mladenig) [reply src/controllers/skillWallet.controller.ts]: [318c7cc](https://github.com/SkillWallet/backend/pull/26/commits/318c7cc53bd84d56dbd4aa80e43fcca083fb14b1)

File: src/controllers/skillWallet.controller.ts (hunk 1 above)
Comment (mladenig) [reply src/controllers/skillWallet.controller.ts]: [318c7cc](https://github.com/SkillWallet/backend/pull/26/commits/318c7cc53bd84d56dbd4aa80e43fcca083fb14b1)

**Pull request merged**